ui/components/ui/textarea/textarea.js

```diff
@@ -38,7 +38,7 @@ const TextArea = ({
       borderColor={COLORS.BORDER_DEFAULT}
       borderRadius={SIZES.SM}
       borderStyle={BORDER_STYLE.SOLID}
-      padding={[4, 4]}
+      padding={4}
       width={BLOCK_SIZES.FULL}
       {...boxProps}
     >
```

Comment:
Updating previous array props for different directions to singular direction props now that array props are reserved for responsive props